app/v2/models/response/createUkPropertyPeriodSummary/CreateUkPropertyPeriodSummaryResponse.scala

```diff
@@ -16,10 +16,28 @@
 
 package v2.models.response.createUkPropertyPeriodSummary
 
+import config.AppConfig
 import play.api.libs.json.{Json, OFormat}
+import v2.hateoas.HateoasLinksFactory
+import v2.models.hateoas.{HateoasData, Link}
 
 case class CreateUkPropertyPeriodSummaryResponse(submissionId: String)
 
 object CreateUkPropertyPeriodSummaryResponse {
   implicit val format: OFormat[CreateUkPropertyPeriodSummaryResponse] = Json.format[CreateUkPropertyPeriodSummaryResponse]
-}
\ No newline at end of file
+  
+  implicit object LinksFactory extends HateoasLinksFactory[CreateUkPropertyPeriodSummaryResponse, CreateUkPropertyPeriodSummaryHateoasData] {
+    override def links(appConfig: AppConfig, data: CreateUkPropertyPeriodSummaryHateoasData): Seq[Link] = {
+
+      /*
+      * Add retrieveUkPropertyPeriodSummary call when it ready (uncomment below 2 lines)
+      * import data._
+      * retrieveUkPropertyPeriodSummary(appConfig, nino, businessId, submissionId)
+      * */
+
+      Seq.empty
```

Comment:
Can we add these in now?

Reply:
is retrieve ready?

Reply:
No but we know what the url will be. Otherwise will have to revisit this endpoint later and fix up its ITs. And we could easily forget to do this.